tests/Kustomaur.Builder.Tests/DashboardFilters/FilterModelTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Kustomaur.Models.Filters;
+using Xunit;
+
+namespace Kustomaur.Builder.Tests.DashboardFilters
+{
+    public class FilterModelTests
+    {
+        [Fact]
+        public void FilterModel_PropertiesAreSetCorrectly()
```

Comment:
Can we also add a test which asserts the JSON that is produced?

Reply:
Edited.